src/components/CheckoutTabs/CheckoutConfirmationTab.tsx

```diff
@@ -0,0 +1,52 @@
+import { Button } from 'components/Buttons/Button';
+import { CheckoutSection } from 'components/CheckoutSection/CheckoutSection';
+import { OrderDetailsMinimal } from 'components/OrderDetails/OrderDetailsMinimal';
+import { CheckoutTabContext, CheckoutTabType } from 'context/CheckoutTabContext';
+import { useLocalOrder } from 'context/OrderContext';
+import { useContext } from 'react';
+import { GBPFormat } from 'utilities/GBPFormat';
+import { SERVICE_CHARGE } from 'utilities/variables';
+import FoodPrepGif from '../../assets/food-prep.gif';
+
+export const CheckoutConfirmationTab = (): React.ReactElement => {
+  const { localOrder } = useLocalOrder();
+  const { setCurrentTab } = useContext(CheckoutTabContext);
+
+  return (
+    <div className="flex h-full w-full flex-col">
+      <div className="flex w-full flex-col items-center justify-center">
+        <img src={FoodPrepGif} className="aspect-square w-[150px]"></img>
+        <h1 className="mb-4 text-[20px] font-[600] text-[#333F4C]">Order received</h1>
+      </div>
+
+      <CheckoutSection title="Order">
+        <div className="flex w-full flex-col gap-2">
+          <OrderDetailsMinimal items={localOrder.lineItems} />
+        </div>
+      </CheckoutSection>
+
+      <CheckoutSection title="Order details">
+        <div className="flex w-full flex-row justify-between border-b-2 text-[12px]">
+          <span>Joe Bloggs</span>
```

Comment:
it's not necessary, but you could make sure you have all your placeholder data in variables for now, that way when it comes to using real data down the line it'll be harder to accidentally miss places that need changing